company_website/static/main_page/testimonials.sass

```diff
@@ -0,0 +1,237 @@
+
+    .nav-button
+        width: 54px
+        height: 51px
+        margin: auto
+
+        &:focus
+            outline: none
+
+    .section_title
+        text-align: center
+        text-transform: uppercase
+        color: $dark-blue
+        font-size: 2.2rem
+        font-weight: 700
+        margin-bottom: 2rem
+
+    .container
+        padding: 0
+        margin: 0
+        max-width: $max-section-width
+        width: 100%
+        align-items: center
+
+        .slideshow
+            display: flex
+
+            .nav-button
+                width: 54px
+                height: 51px
+                margin: auto
+
+                &-prev
+                    @extend .nav-button
+                    margin-left: 0
+                    float: left
+
+                &-next
+                    @extend .nav-button
+                    float: right
+                    margin-right: 0
+
+                &:focus
+                    outline: none
+
+            @media only screen and (max-width: $nav-button-breakpoint)
+                    .nav-button
+                        display: none
+
+            .slides
+                float: None
+                margin: auto
+                max-width:1172
+
+                .testimonial
+                    max-width: 1164px
+                    min-height: 309px
+                    height: 100%
+                    border-radius: 1.5rem
+                    position: relative
+                    overflow: hidden
+                    display: flex
+                    padding: 4.6% 5.5% 0 3%
+                    margin: 0 4px
+                    background-color: $blue
+
+                    .background-image
+                        width: 100%
+                        height: auto
+                        bottom: 0
+                        left: 0
+                        z-index: 0
+
+                    .profile-picture
+                        position: relative
+
+                        .image-cropper
+                            width: 140px
+                            height: 140px
+                            position: relative
+                            overflow: hidden
+                            border-radius: 50%
+                            margin: 0 auto
+
+                            img
+                                display: inline
+                                margin: 0 auto
+                                height: auto
+                                width: 100%
+
+                    .opinion-section
+                        display: flex
+                        position: relative
+                        text-align: center
+                        margin-left: 5.1%
+                        float: right
+
+                        .quote-mark
+                            width: 45px
+                            height: 39px
+
+                        .text-area
+                            margin: -6 30 0
+                            font-size: 21
```

Comment:
Here and in other few places you forgot to add what kind of value it is: `%` `px` or something else.

Reply:
Good point. 👍  I tried to keep it consistent and update if necessary, but I must've missed few of them. I'll look over again and fix it.

Reply:
Done.